docs/dev/design-decisions.md

```diff
@@ -142,52 +146,81 @@ glance   True     Setup complete
 
 $ oc get GlanceAPI
 
-NAME              NETWORKATTACHMENTS   STATUS   MESSAGE
-glance-external                        True     Setup complete
-glance-internal                        True     Setup complete
+NAME                      NETWORKATTACHMENTS   STATUS   MESSAGE
+glance-default-external                        True     Setup complete
+glance-default-internal                        True     Setup complete
 ```
 
-and they can be inspected as regular k8s objects.
-However, it's possible to not split the deployment between internal and external
-API deployment, and reduce everything to a single API instance.
-To do that, a Glance CR spec like the following should be created:
+The resulting CRs can be inspected as regular k8s objects.
+The following diagram describes how k8s resources/object are connected.
 
 ```
-...
+                                                      [statefulset(s)]
+                                                +-------------------------+
++------------+       +------------------+ |---> | glance-default-internal |
+| glance CR  | |---> | glance-default CR|       +-------------------------+
```

Comment:
Maybe `glance-default CR` should be `glanceAPI-default CR` to clearly indicate that it's a `GlanceAPI` CR?

Reply:
Makes sense, it also helps to properly identify the objects created. Thanks, I updated it!